frontend/packages/rhoas-plugin/src/topology/rhoas-topology-plugin.tsx

```diff
@@ -0,0 +1,67 @@
+import { Plugin } from '@console/plugin-sdk';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ALLOW_SERVICE_BINDING_FLAG } from '@console/topology/src/const';
+import { getExecutableCodeRef } from '@console/dynamic-plugin-sdk/src/coderefs/coderef-utils';
+import {
+  TopologyComponentFactory,
+  TopologyCreateConnector,
+  TopologyDataModelFactory,
+} from '@console/topology/src/extensions/topology';
+import { getRhoasComponentFactory, getRhoasTopologyDataModel } from './index';
+import { WatchK8sResources } from 'public/components/utils/k8s-watch-hook';
+import { KafkaConnectionModel } from '../models';
+import { FLAG_RHOAS } from '../const';
+
+export type TopologyConsumedExtensions =
+  | TopologyComponentFactory
+  | TopologyDataModelFactory
+  | TopologyCreateConnector;
+
+const getRhoasWatchedResources = (namespace: string): WatchK8sResources<any> => {
```

Comment:
Better to have type instead of any. Or can we use `WatchK8sResource`  as we are fetching only one resource here?
```suggestion
const getRhoasWatchedResources = (namespace: string): WatchK8sResources<{ kafkaConnections: K8sResourceKind[] }> => {
```